### DISCUSSION TITLE
Installed with one-click install on Windows, but when I launch it... nothing happens?

### DISCUSSION THREAD

**Discussion by Exo1943** [oobabooga/text-generation-webui#3270] [Q&A] [answered]
I have Windows and an NVIDIA graphics card.  I selected NVIDIA when asked.  I have installed three times, but nothing ever opens.

This window opens, but nothing happens after.

![Untitled](https://github.com/oobabooga/text-generation-webui/assets/140352176/5ec4b525-8d2a-45c4-aa9c-b4159756b793)

**Comment by viperwasp** [comment, marked as answer]
Two things. 
1. You need to download any model and put it in the models folder. Different models have different correct ways of running them. 
2. You still need to copy and paste http://127.0.0.1:7860 into a web browser after you launch this for the WEB UI to open. 

I think you can do either of these steps out of order but yes you have to launch the WEB UI it does not open automatically.

**Reply by Exo1943** [reply]
Oh ok.  I got it.  Was used to Kobold launching my browser for me.  Is it possible to connect to my host machine over LAN by connecting to its local IP address from another machine, like in Kobold?  I would like to have one machine host the AI, but utilize it with a different machine on the same LAN.

**Reply by viperwasp** [reply]
Sorry all of that is outside of how I use any of this. No idea. Hopefully someone else can answer.

**Reply by missionfloyd** [reply]
> Is it possible to connect to my host machine over LAN by connecting to its local IP address from another machine, like in Kobold?

Edit `webui.py` and add `--listen` on [line 15](https://github.com/oobabooga/one-click-installers/blob/cc2ed46d44cb8aed982c2a13ccfe6b27c06b6068/webui.py#L15). E.g., `CMD_FLAGS = '--chat --listen'`

**Reply by Exo1943** [reply]
This works, thank you.
